styles/globals.css

```
@@ -46,3 +46,94 @@ main {
   font-size: 24px;
   line-height: 29px;
 }
+
+.card {
+  width: 350px;
+  height: 500px;
+  background-color: #4d1c22;
+  border-radius: 15px;
+  position: relative;
+}
+
+.card-image {
+  background-image: url(../assets/Long\ March\ 3BE\ ChinaSat\ 26.jpg);
```

Cambienmos el nombre del image para que no tenga espacios, podría ser
```suggestion
  background-image: url(../assets/dummy_card_cover.jpg);
```